src/components/cart/CartProduct.tsx

```diff
@@ -0,0 +1,234 @@
+import { useState } from 'react';
+import { useRecoilState } from 'recoil';
+import styled from 'styled-components';
+
+import { CheckCheckedIcon, CheckNotCheckedIcon, MinusIcon, PlusIcon } from '../../assets/icon';
+import { overallCheckState, overallQuantityState } from '../../states/cart';
+import { CartProductsData } from '../../types/cart';
+import { patchCartQuantityData } from '../../utils/lib/cart';
+import ProductTag from '../common/ProductTag';
+
+interface CartProductProp {
+  cartProduct: CartProductsData;
+}
+
+const CartProduct = (props: CartProductProp) => {
+  const { cartProduct } = props;
+  const cartProductId = cartProduct.cartProductId;
+
+  // 전역상태: 모든 상품의 체크 여부 정보, 모든 상품의 수량 정보
+  const [overallCheck, setOverallCheck] = useRecoilState(overallCheckState);
+  const [overallQuantity, setOverallQuantity] = useRecoilState(overallQuantityState);
+
+  // 지역상태: 이 상품의 원래 가격, 이 상품의 할인 가격
+  const [calcOriginalPrice, setCalcOriginalPrice] = useState(
+    cartProduct.originalPrice * overallQuantity[cartProductId],
+  );
+  const [calcDiscountPrice, setCalcDiscountPrice] = useState(
+    cartProduct.discountPrice * overallQuantity[cartProductId],
+  );
+
+  const patchCartQuantity = async (change: boolean) => {
+    try {
+      const {
+        data: { data },
+      } = await patchCartQuantityData({ cartProductId: cartProductId, changeStatus: change });
```

Comment:
```suggestion
      } = await patchCartQuantityData({ cartProductId, changeStatus: change });
```
<a href="https://poiemaweb.com/es6-enhanced-object-property">축약형 프로퍼티</a> 가능!